src/coreclr/vm/jitinterface.cpp

```diff
@@ -13030,9 +13042,16 @@ PCODE UnsafeJitFunction(PrepareCodeConfig* config,
     MethodDesc* ftn = nativeCodeVersion.GetMethodDesc();
 
     PCODE ret = NULL;
+    int64_t jitStartTimestamp = 0;
+    int64_t jitEndTimestamp = 0;
+    int64_t jitTimeQPCTicks = 0;
+    LARGE_INTEGER qpcValue;
 
     COOPERATIVE_TRANSITION_BEGIN();
 
+    QueryPerformanceCounter(&qpcValue);
+    jitStartTimestamp = static_cast<int64_t>(qpcValue.QuadPart);
```

Comment:
Nit: You can save this copy by defining `jitStartTimestamp` and `jitEndTimestamp` as LARGE_INTEGER; and do the `static_cast` only once you need to subtract the two.